service/controller/app/v1/resource/chartcrd/create.go

```diff
@@ -0,0 +1,59 @@
+package chartcrd
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1"
+	"github.com/giantswarm/app-operator/service/controller/app/v1/controllercontext"
+	"github.com/giantswarm/app-operator/service/controller/app/v1/key"
+	"github.com/giantswarm/backoff"
+	"github.com/giantswarm/microerror"
+)
+
+// EnsureCreated ensures the chart CRD exists in the tenant cluster so the
+// chart resource can create chart CRs.
```

Comment:
This is the new resource. It runs before the chart resource so the chart CRD is always present in the tenant cluster.